<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>645</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_name">
     <widget class="QLineEdit" name="lineEdit_userName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>your name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_submitName">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_menu">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 200, 165);
color: rgb(55, 50, 40);</string>
     </property>
     <widget class="QPushButton" name="pushButton_single">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 155, 80)</string>
      </property>
      <property name="text">
       <string>Singleplayer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_multi">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 155, 80)</string>
      </property>
      <property name="text">
       <string>Multiplayer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_host">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 155, 80)</string>
      </property>
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_join">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 155, 80)</string>
      </property>
      <property name="text">
       <string>Join</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Hiragino Kaku Gothic Std</family>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GOMOKU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_join">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 200, 165);
color: rgb(55, 50, 40);</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_serverAddress">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>ip address</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_joinServer">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 155, 80)</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 155, 80)</string>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_game">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>605</y>
        <width>81</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 200, 165);
color: rgb(55, 50, 40);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_player1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>605</y>
        <width>81</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>player 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_player2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>605</y>
        <width>81</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>player 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_blackCircle">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>615</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>⚫</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_whiteCircle">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>614</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;⚪&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
